files/usr/sbin/install-nx.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+if [ $# -eq 0 ]; then
+    echo -e "No arguments supplied, target device for installation needed\n$0 -t /dev/sdX [-s source]"    
+    exit 1
+fi
+while getopts "ft::s::" opt; do
+            case $opt in
+            (f) F=1 ;; #Force write
+            (s) S=${OPTARG} ;; #Source image
+            (t) T=${OPTARG} ;; #Target disk
```

Comment:
Please use variables with one chars only on limited scope, like inside loops.
Generally speaking variable names should explain the purpose.
Also, as convention, variables in upper case are usually constant.

So the above code should be something like:
```
            (f) force=1 ;; #Force write
            (s) source=${OPTARG} ;; #Source image
            (t) target=${OPTARG} ;; #Target disk
```